server/public/index.go.html

```diff
@@ -17,7 +17,39 @@
     <script src="{{ rev "/auditorium/vendor.js" }}"></script>
     <script src="{{ rev "/auditorium/index.js" }}"></script>
     <noscript>
-      {{ __ "Please enable JavaScript in your browser's settings to use this application." }}
+      <div class="f5 roboto dark-gray">
+        <div class="w-100 h3 bg-black-05">
+          <div class="mw8 center flex ph3 pt2">
+            <img src="/offen-icon-black.svg" alt="Offen logo" width="37" height="40" class="ma0 mt1 mr3">
+            <h1 class="f2 normal ma0 mt1">Offen</h1>
+          </div>
+        </div>
+        <div class="mw8 center hp0 ph3-ns">
+          <p class="dib pa2 br2 ma0 mt3 ml3 ml0-ns mr3 mr0-ns bg-light-yellow">
+            It seems like your browser is <strong>blocking JavaScript.</strong>
+          </p>
+          <div class="w-100 ph3 ph4-ns pv4 mt4 mb4 bt bb ba-ns br0 br2-ns b--black-10 bg-white">
+            <div class="w-100 w-90-ns mr3">
+              <h3 class="f5 b ma0 mb2">Offen is a fair and open source analytics software. It is installed on the website that linked you here.</h3>
+              <p class="ma0">For Offen to work, your browser has to accept JavaScript and cookies. Please change your browser settings and reload this page. Subsequently, you will be asked for consent to collect your usage data.</p>
```

Comment:
I'm not entirely sure if we need this, but as-is this portion of text would always be in English, no matter if the application is configured to use a different locale. Should we add translations here too?

Reply:
in a finished product yes